File: Iniciante/Pagina 03/1132/1132.c
#include <stdio.h>
 
int main() {
 
    int a, b, inicio, fim, total = 0;

    scanf("%d", &a);
    scanf("%d", &b);

    if (a > b) {
        fim = a;
        inicio = b;
    } else {
        fim = b;
        inicio = a;
    }

    for (; inicio <= fim; inicio++) {
        if (inicio % 13 != 0) {
            total += inicio;
        }
    }

    printf("%d\n", total);
 
    return 0;
}

File: Iniciante/Pagina 03/1132/1132.py
total = 0
a = int(input())
b = int(input())

if a > b:
    fim = a + 1
    inicio = b
else:
    fim = b + 1
    inicio = a

for inicio in range(inicio, fim):
    if (inicio % 13 != 0):
        total += inicio

print(total)
